core/services/functions/listener.go

```diff
@@ -488,3 +508,18 @@ func (l *FunctionsListener) timeoutRequests() {
 		}
 	}
 }
+
+func (l *FunctionsListener) reportSourceCodeDomains(requestId [32]byte, domains []string) {
+	r := &telem.FunctionsRequest{
+		RequestId: requestId[:],
+		Domains:   domains,
+	}
+
+	bytes, err := proto.Marshal(r)
+	if err != nil {
+		l.logger.Warnw("telem.FunctionsRequest marshal error", "err", err)
+	} else {
+		l.logger.Debugw("Reporting domains", "requestID", formatRequestId(requestId), "domains", domains)
```

Comment:
I wonder if it makes sense to log all domains here... This can be very large and pollute logs. Also I guess it's not very helpful to have this in core node logs as it's going out to telemetry collection anyway.

Reply:
Agreed. Removed.